moveit_ros/warehouse/warehouse/src/warehouse_connector.cpp

```diff
@@ -36,12 +36,25 @@
 
 #include <sys/types.h>
 #include <signal.h>
-#include <unistd.h>
 #include <rclcpp/rclcpp.hpp>
 #include <moveit/warehouse/warehouse_connector.h>
 
 static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit.ros.warehouse.warehouse_connector");
 
+#ifdef _WIN32
```

Comment:
My approach here was to ifdef out the whole function so it does nothing but return false:
https://github.com/ms-iot/moveit2/commit/b56eba13e5fb7ab594f774b0b51cdd8f03dad5e2

I like adding the error message to make it clear what went wrong.